clients/ros2_client/geoscenario_msgs/msg/Tick.msg

```diff
@@ -3,8 +3,8 @@ std_msgs/Header header
 uint32 tick_count
 float64 simulation_time         # [s] Time passed since the beginning of the simulation
 float64 delta_time              # [s] Time passed since last tick
-geographic_msgs/GeoPoint origin # [deg, deg, m], in WGS84 (EPSG: 4326)
-                                # origin=(0,0,0) indicates that WGS84 coordinates are used
+geographic_msgs/GeoPoint origin # [deg, deg, m, m2], in WGS84 (EPSG: 4326)
```

Comment:
that's not actually `m^2`, just meters because it's the length of the side of the square...